website/docs/api/top-level.mdx

```diff
@@ -340,6 +340,128 @@ use with the `manual=True` argument in `displacy.render`.
 | `options`   | Span-specific visualisation options. ~~Dict[str, Any]~~             |
 | **RETURNS** | Generated entities keyed by text (original text) and ents. ~~dict~~ |
 
+### Visualizer data structures {id="displacy_structures"}
+
+You can also use displaCy's data format to manually render data. This can be
+useful if you want to visualize output from other libaries. You can find
+examples of displaCy's data format on the
+[usage page](/usage/visualizers#manual-usage).
```

Comment:
Do you think we should still link to the Usage page when the same examples are now in the sidebar? I think it might be more confusing to a user that they would be reading and click this link to be brought away from the documentation.

Reply:
I agree! I'll remove the link

Reply:
Removed the link 😇